mimic/canned_responses/maas_test_check.py

```diff
@@ -0,0 +1,67 @@
+"""
+Canned response for /test-check
+"""
+
+import collections
+
+
+def test_check(checkType):
+    """
+    Support test-check API
+    """
+    test_check_responses = collections.defaultdict(lambda: "")
+    test_check_responses["agent.cpu"] = [
+        {
+            "timestamp": 0,
+            "available": True,
+            "status": "success",
+            "metrics": {
+                "user_percent_average": {
+                    "data": 7.1695253641707,
```

Comment:
Right now, `MimicCore` holds on to the one object which generates I/O events - timeouts - within mimic; `MimicCore.clock`.  We could expand this to also hold on to a `random` instance, and the code in question here could do something like `core.rng.uniform(MIN_CPU_VAL, MAX_CPU_VAL)`.